### ISSUE TITLE
Gun Gun Pixies

### ISSUE THREAD

**Issue by untouchable6849** [skyline-emu/skyline-games-list#149]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

#1064

### Android Version

12

### SoC type

Snapdragon

### Device GSMArena link

https://m.gsmarena.com/samsung_galaxy_z_fold2_5g-10342.php

### Game Behaviour

![Screenshot_20220501-082110_Skyline](https://user-images.githubusercontent.com/78664886/166135573-d2d34375-87ac-4a65-b1e5-a3ebadb94307.jpg)
![Screenshot_20220501-082114_Skyline](https://user-images.githubusercontent.com/78664886/166135574-5c03c334-1ada-4a39-840a-ffc6b4eecb94.jpg)
![Screenshot_20220501-082121_Skyline](https://user-images.githubusercontent.com/78664886/166135575-ddc96ba8-bc56-49dc-b912-69f84dd1e6fe.jpg)
![Screenshot_20220501-082123_Skyline](https://user-images.githubusercontent.com/78664886/166135576-7bbcea50-31b0-445d-b69e-3f9fa254eb57.jpg)

• after intro, crash

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[Gun Gun Pixies.txt](https://github.com/skyline-emu/skyline-games-list/files/8598425/Gun.Gun.Pixies.txt)

**Comment by koukichi7010** [comment]
W1524HOS-0HandleRequest: Cannot find function in service 'visrv::IApplicationDisplayService': 0x836 (2102)

W1527GPFIFOSetProvokingVertex: Cannot set provoking vertex to last without host GPU support

**Comment by iniMerchant64** [comment]
E20751HOS-1Failed to read from fd: Bad address

**Comment by koukichi7010** [comment]
You should've just edited the issue and ping Bylaws to change the tags than commenting another one

**Comment by Switchpirate1** [comment]
I'm using the same build, but on poco F1 (sd845), facing issues with graphics

**Comment by DanXG790** [comment]
@Switchpirate1 maybe it is the Adreno Driver version of your device, it still works the same in the latest build, but also try to use the Handled mode for less bugs

**Comment by Switchpirate1** [comment]
Is there any way to change the driver version?

**Comment by DanXG790** [comment]
@Switchpirate1 yes, but need have rooted your device, search in xda developers the thread [(here)](https://forum.xda-developers.com/t/driver-10-0-adreno-600-series-opengl-vulkan-drivers-v-0481-0-1-1-128.4089967/)  and the file that works for you Adreno 630, that's all I know since I've never tried.

**Comment by ikhoeyZX** [comment]
![Screenshot_20231217_012110](https://github.com/strato-emu/compat-list/assets/79812109/fbd5d4c1-f9df-43ce-aefa-ae994da674b3)
playable
